tests/unit_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -159,6 +172,36 @@ def test_get_deactivated_participants(self,
 
         self.assertEqual(expected_response, dataset_response)
 
+    @mock.patch('utils.participant_summary_requests.store_participant_data')
+    @mock.patch(
+        'utils.participant_summary_requests.get_site_participant_information')
+    def test_get_site_participant_information(
+        self, mock_get_site_participant_information,
+        mock_store_participant_data):
+
+        # Pre conditions
+        mock_get_site_participant_information.return_value = self.fake_participant_info_dataframe
+
+        # Tests
+        dataframe_response = psr.get_site_participant_information(
+            self.project_id, self.fake_hpo)
+
+        dataset_response = psr.store_participant_data(dataframe_response,
+                                                      self.project_id,
+                                                      self.destination_table)
+
+        expected_response = mock_store_participant_data(dataframe_response,
+                                                        self.project_id,
+                                                        self.destination_table)
+
+        # Post conditions
+        pandas.testing.assert_frame_equal(
+            dataframe_response,
+            pandas.DataFrame(self.updated_site_participant_info,
+                             columns=psr.FIELDS_OF_INTEREST_FOR_VALIDATION))
+
+        self.assertEqual(expected_response, dataset_response)
```

Comment:
Not sure you need this stuff either.

Reply:
I just now realized that this is for the ticket where we store the dataframe so I jumped the gun here 😅